pkg/virt-controller/services/template.go

```diff
@@ -370,6 +370,25 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 			},
 		})
 	}
+
+        // Fix for kind "same host" migration, how to do it right ?
+        // keep in mind that the creation of the proudct_uuid i still ran manually
+	volumeMounts = append(volumeMounts, k8sv1.VolumeMount{
```

Comment:
I think @maiqueb already wrote it and I totally agree. I wouldn't touch the code base of all the virt-launchers only for fixing a kind bug. This should be done as part of the make cluster-sync, only for kind provider.

Reply:
I would add it to the tests. Would create a migratable vm that all the migration tests can use (in case it is a kind cluster).

Reply:
but at the end we understood that it need to be here, but later on now there is maybe a way that we can research

on the other hand , there were couple people that do thought its good on kubevirt
anyway i am focusing a moment for a research

the bug is not Kind ,but non robustness of kubevirt for containers, this is a side note

Reply:
> I would add it to the tests. Would create a migratable vm that all the migration tests can use (in case it is a kind cluster).

i need it in the virt-launchers, i will investigate first why virtctl migrate works
and from tests it doesnt, and we will see after